mapviz_plugins/src/odometry_plugin.cpp

```diff
@@ -377,6 +422,17 @@ namespace mapviz_plugins
       ui_.arrow_size->setValue(node["arrow_size"].as<int>());
     }
 
+    if (node["show_timestamps"])
```

Comment:
This does not appear to have anything to do with showing covariances.

Please constrain changes in a single PR to a single issue.